src/uct/base/uct_cm.c

```diff
@@ -51,6 +51,32 @@ ucs_status_t uct_cm_config_read(uct_component_h component,
     return UCS_OK;
 }
 
+ucs_status_t uct_cm_check_ep_params(const uct_ep_params_t *params)
+{
+    if (!(params->field_mask & UCT_EP_PARAM_FIELD_CM)) {
+        ucs_error("UCT_EP_PARAM_FIELD_CM is not set. field_mask 0x%lx",
+                  params->field_mask);
+        return UCS_ERR_INVALID_PARAM;
+    }
+
+    if (!(params->field_mask & UCT_EP_PARAM_FIELD_SOCKADDR_CB_FLAGS) ||
+        !(params->sockaddr_cb_flags & UCT_CB_FLAG_ASYNC)) {
+        ucs_error("UCT_EP_PARAM_FIELD_SOCKADDR_CB_FLAGS and UCT_CB_FLAG_ASYNC "
```

Comment:
makes sense to me to print `params->field_mask` and `params->sockaddr_cb_flags` as you did for `params->field_mask` above and below, wdyt?